### ISSUE TITLE
tumor area annotated by XML is too small in sample 28 and 45

### ISSUE THREAD

**Issue by funger1990** [wisepaip/paip2020#6]
Tumor area annotated by XML is too small in some cases, such as training data 28 and 45. Is XML annotation the ground truth for tumor?

**Comment by funger1990** [comment]
I re-read the description on the downloading web. It's written that "The organizers would like to clarify that the XML File is only for reference and is not the ground truth. Image masks provided as TIF files are the ground truth."

But I can't find "image masks provided as TIF files" for tumor area. I am kind of confused.

**Comment by Paip-2020** [comment]
Dear challenger,
Thank you for your inquiry.
We fully understand your confusion. That sentence "Image masks provided as TIF files are the ground truth." is not proper for the PAIP2020 challenge policy. We have just removed the expression from the web page. 
We note that we would like to clarify that the XML File is only for reference and is not the ground truth.

Also, in some case, the tumor area may be small as you mentioned, but we believe these slides are also necessary as part of the training because it can actually happen in the medical field. Considering these points, we would appreciate that you suggest best model for us.
